wger/config/nginx.conf

```
@@ -0,0 +1,27 @@
+upstream wger {
+    server web:8000;
```

Use full service name.
```suggestion
    server wger_web_1:8000;
```